### DISCUSSION TITLE
Sign in with Apple via the In App Pop Up

### DISCUSSION THREAD

**Discussion by deliqs** [supabase/supabase#7472] [Questions] [answered]
Hi,

I cannot find if it is possible to sign in with Apple via the In App Pop Up (or whatever it is called) via the Supabase Dart package.

Here is a screenshot of what I mean.
![image](https://user-images.githubusercontent.com/88924253/176195652-2dcb141d-0322-4bfa-8d86-0fc910ef7431.png)

**Comment by dshukertjr** [comment, marked as answer]
Hi @deliqs,

We are working on this feature at the moment hoping to bring it by August. You can watch [this](https://github.com/supabase-community/supabase-flutter/issues/5) issue to see the progress.

**Reply by deliqs** [reply]
Thanks for pointing me to the issue!
